yewprint-doc/src/slider/example.rs

```diff
@@ -90,24 +94,26 @@ impl Component for Example {
                     values={percentage_labels}
                     selected={self.integer}
                     intent={ctx.props().intent}
-                    value_label={Cow::Owned(format!("{}%", self.integer))}
+                    value_label={IString::from(format!("{}%", self.integer))}
```

Comment:
Already converted for you. That's the whole point of AttrValue

```suggestion
                    value_label={format!("{}%", self.integer)}
```